### ISSUE TITLE
Org chart: Application hangs attempt to navigate to root parent

### ISSUE THREAD

**Issue by BethRR** [winwinwiki/winwin-frontend#689]
I believe this is related to the migration where I added parent_id and rootParent_id so that these would be visible in the Kibana Index ( per discussion with Arvind) on Weds night. I think the code is confused by a couple aspects of the rootparent ids.

**Two sets of steps to repro to help illustrate full detail of problem:**
Set1: 
1) Navigate to https://stage.winwin.care/organizations/1 (NOTE: you CANNOT navigate via org [management due to a potentially related issue](https://github.com/winwinwiki/winwin-frontend/issues/690))
2) click on org chart. Note that the app hangs and nothing renders.
In the database: id 1 has a parent_id of 1 and a root_parent_id of 1

Set 2 
1) Navigate to https://stage.winwin.care/organizations/4771
2) click on org chart and  Note that it renders. 
3) Click the up button to navigate to its parent. Note that it correctly renders with parent org is Veterans Integrated Service Network (4756)
4) Click the up button to navigate to the next parent level. Note that the app hangs and nothing renders. 
In the database: id 4771 has a parent_id of 4756 and a root_parent_id of 4569 (these are correct). However, 4756 has a parent (id 4614 correct in the database) and a root parent of 4569.
In other words the hierarchy for id 4771 from leaf to root is : 4771 | 4756 | 4614 | 4569
**The error:** 

https://stage.winwin.care/organizations/4771
**Here is the error in the console:**
GET https://backend-stage.winwin.care/organization/1/suborganization 500

Timestamp: "2019-10-18T22:50:18.868+0000", status: 500, error: "Internal Server Error",…}
error: "Internal Server Error"
message: "No message available"
path: "/organization/1/suborganization"
status: 500
timestamp: "2019-10-18T22:50:18.868+0000"

{timestamp: "2019-10-18T23:23:40.733+0000", status: 500, error: "Internal Server Error",…}
error: "Internal Server Error"
message: "No message available"
path: "/organization/4569/suborganization"
status: 500
timestamp: "2019-10-18T23:23:40.733+0000"

**Comment by arvindkhatik012** [comment]
@BethRR As discussed with you the root cause of the issue on daily stand up call, I am marking the issue as fixed
